src/routes/SafeAppLandingPage/SafeAppLandingPage.tsx

```diff
@@ -0,0 +1,312 @@
+          </LoaderContainer>
+        ) : (
+          <>
+            {/* Safe App details */}
+            {safeAppDetails && (
+              <SafeAppDetails
+                iconUrl={safeAppDetails.iconUrl}
+                name={safeAppDetails.name}
+                description={safeAppDetails?.description}
+                availableChains={availableChains}
+              />
+            )}
+
+            <ActionsContainer>
+              <UserSafeContainer>
+                <Title size="xs">Use the dApp with your Safe!</Title>
+                {isWalletConnected ? <CreateNewSafe /> : <ConnectWallet />}
+              </UserSafeContainer>
+
+              {/* Demo Safe Section */}
+              <TryDemoSafe safeAppUrl={safeAppUrl} />
+            </ActionsContainer>
+          </>
+        )}
+      </StyledCard>
+    </Container>
+  )
+}
+
+export default SafeAppLandingPage
+
+const SafeAppDetails = ({ iconUrl, name, description, availableChains }) => {
+  const showAvailableChains = availableChains && availableChains.length > 0
+
+  return (
+    <>
+      <DetailsContainer>
+        <SafeIcon src={iconUrl} />
+        <DescriptionContainer>
+          <SafeAppTitle size="sm">{name}</SafeAppTitle>
+          <div>{description}</div>
+        </DescriptionContainer>
+      </DetailsContainer>
+      <Separator />
+
+      {/* Available chains */}
+      {showAvailableChains && (
+        <>
+          <ChainLabel size="lg">Available networks</ChainLabel>
+          <ChainsContainer>
+            {availableChains.map((chainId) => (
+              <div key={chainId}>
+                <NetworkLabel networkInfo={getChainById(chainId)} />
+              </div>
+            ))}
+          </ChainsContainer>
+          <Separator />
+        </>
+      )}
+    </>
+  )
+}
+
+const CreateNewSafe = () => {
+  return (
+    <>
+      <BodyImage>
+        <Img alt="Vault" height={92} src={SuccessSvg} />
+      </BodyImage>
+
+      <Button size="lg" color="primary" variant="contained" component={Link} to={OPEN_SAFE_ROUTE}>
+        <Text size="xl" color="white">
+          Create new Safe
+        </Text>
+      </Button>
+    </>
+  )
+}
+
+const ConnectWallet = () => {
+  const { clickAway, open, toggle } = useStateHandler()
+
+  return (
+    <StyledProvider>
+      <Provider
+        info={<ProviderDisconnected />}
+        open={open}
+        toggle={toggle}
+        render={(providerRef) =>
+          providerRef.current && (
+            <StyledPopper
+              anchorEl={providerRef.current}
+              open={open}
+              placement="bottom"
+              popperOptions={{ positionFixed: true }}
+            >
+              <ClickAwayListener onClickAway={clickAway} touchEvent={false}>
+                <List component="div">
+                  <ConnectDetails />
+                </List>
+              </ClickAwayListener>
+            </StyledPopper>
+          )
+        }
+      />
+    </StyledProvider>
+  )
+}
```

Comment:
Can this be reused in the header as well?

Reply:
We are already using all components from the header.

Reply:
I mean, can you use this component in the header instead of duplicating it?